src/App.js

```diff
@@ -1,11 +1,9 @@
-import './App.css';
+import Calculator from './component/calculator';
 
 function App() {
   return (
     <div className="App">
-      <div className="content">
-        <h1>Hello! At</h1>
-      </div>
+      <Calculator />
     </div>
   );
 }
```

Comment:
- [ ] As stated previously by the previous reviewer, kindly make sure the App.js component is class-based instead of a function component.